### DISCUSSION TITLE
Logo *and* title

### DISCUSSION THREAD

**Discussion by uncouthregions** [jpanther/congo#306] [General]
I just added a logo, only to find that my title text no longer displays. Is this a feature of the theme? If so, would it be possible to have both visible at once? I'm willing to try and do this myself, but thought it would be worth mentioning here first.

**Comment by jpanther** [comment]
While this is not something that is currently possible in the default templates, it is possible to provide your own header layout that can add the functionality you need. The simplest workaround would be to copy the header template you like from `/themes/congo/partials/header/[template].html` to your project directory at `/partials/header/custom.html`.

Then edit your custom template and copy the `.Site.Title` link inside the `if` statement with the logo at the top of the template.

To make this simpler, I'll look to make this a configurable option in the theme in a future release.
